src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -95,34 +93,61 @@ public void setup() throws InterruptedException, ServiceBusException, JSONExcept
             .atMost(60, TimeUnit.SECONDS)
             .pollDelay(2, TimeUnit.SECONDS)
             .until(() -> isExceptionRecordCreated(randomPoBox));
+    }
+
+    @Test
+    public void should_atatch_exception_record_to_the_existing_case() throws JSONException {
+        //given
+        Envelope newEnvelope = updateEnvelope("envelopes/new-envelope.json");
+        caseDetails = ccdCaseCreator.createCase(newEnvelope);
+
+        // when
+        callCallbackUrlToAttachExceptionToCase();
 
+        //then
+        await("Exception record is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(this::isExceptionRecordAttachedToTheCase);
+        verifyExistingCaseIsUpdatedWithExceptionRecordData();
     }
 
     @Test
-    @Disabled
-    public void should_atatch_exception_record_to_the_existing_case() {
+    public void should_atatch_exception_record_to_the_existing_case_with_evidence_documents() throws JSONException {
+        //given
+        Envelope newEnvelope = updateEnvelope("envelopes/new-envelope-with-evidence.json");
+        caseDetails = ccdCaseCreator.createCase(newEnvelope);
 
         // when
+        callCallbackUrlToAttachExceptionToCase();
+
+        //then
+        await("Exception record is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(this::isExceptionRecordAttachedToTheCase);
+
+        verifyExistingCaseIsUpdatedWithExceptionRecordData();
+    }
+
+    public void callCallbackUrlToAttachExceptionToCase() {
+        Map<String, Object> caseData = exceptionRecord.getData();
```

Comment:
avoid state in the test and pass parameters for clarity

Reply:
Updated the test to pass the parameters